lib/index.js

```diff
@@ -1,5 +1,5 @@
-import * as IssuerService from "./services/issuer.service";
-import * as VerificationService from "./services/verification.service";
+import * as IssuerService from "./services/issuer.service.js";
```

Comment:
Why should you mention the extension?

Reply:
since this is a ESM library we have to import using '.js' extension. Else we have to use bundlers like Webpack/Parcel/Vite for the build process.